events/ME_Austria_Events.txt

```diff
@@ -1,36 +1,2653 @@
+		
+		add_estate_loyalty = {
+			estate = all
+			loyalty = 10
+		}
+		change_government_reform_progress = 20
+		add_years_of_income = 0.6
+	}
+	
+	option = {
+		name = me_austria_missions.48.b
+		
+		
+	}
+}
+
+country_event = { #Nobility Army
+	id = me_austria_missions.49
+	title = me_austria_missions.49.t
+	desc = me_austria_missions.49.d
+	
+	trigger = {
+		has_country_flag = me_hab_nobles_enlightenment
+		estate_loyalty = {
+			estate = estate_nobles
+			loyalty = 30
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 120
+	}
+	
+	immediate = {
+		clr_country_flag = me_hab_nobles_enlightenment
+	}
+	
+	option = {
+		name = me_austria_missions.49.a
+		
+		create_general = { tradition = 60 }
+		add_army_tradition = 10
+		set_country_flag = me_hab_destroying_nobility_rights
+	}
+}
+
+country_event = { #Getting rid of the many nobility rights
+	id = me_austria_missions.50
+	title = me_austria_missions.50.t
+	desc = me_austria_missions.50.d
+	
+	trigger = {
+		has_country_flag = me_hab_destroying_nobility_rights
+		estate_loyalty = {
+			estate = estate_nobles
+			loyalty = 30
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 120
+	}
+	
+	immediate = {
+		clr_country_flag = me_hab_destroying_nobility_rights
+	}
+	
+	option = {
+		name = me_austria_missions.50.a
+		
+		add_country_modifier = {
+			name = me_hab_centralized_nobility
+			duration = 7300
+		}
+		set_country_flag = me_hab_create_a_centralized_state
+	}
+}
+
+country_event = { #A centralized state
+	id = me_austria_missions.51
+	title = me_austria_missions.51.t
+	desc = me_austria_missions.51.d
+	
+	trigger = {
+		has_country_flag = me_hab_create_a_centralized_state
+		estate_loyalty = {
+			estate = estate_nobles
+			loyalty = 30
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 120
+	}
+	
+	immediate = {
+		clr_country_flag = me_hab_create_a_centralized_state
+	}
+	
+	option = {
+		name = me_austria_missions.51.a
+		
+		if = {
+			limit = {
+				mil = 6
+			}
+			add_mil_power = 100
+		}
+		else = {
+			change_mil = 1
+		}
+		if = {
+			limit = {
+				adm = 6
+			}
+			add_adm_power = 100
+		}
+		else = {
+			change_adm = 1
+		}
+		if = {
+			limit = {
+				dip = 6
+			}
+			add_dip_power = 100
+		}
+		else = {
+			change_dip = 1
+		}
+		
+		set_country_flag = me_hab_allow_edict_of_tolernace
+	}
+}
+
+country_event = { #Edict of Tolerance
+	id = me_austria_missions.52
+	title = me_austria_missions.52.t
+	desc = me_austria_missions.52.d
+	
+	trigger = {
+		has_country_flag = me_hab_allow_edict_of_tolernace
+		estate_loyalty = {
+			estate = estate_nobles
+			loyalty = 30
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 120
+	}
+	
+	immediate = {
+		clr_country_flag = me_hab_allow_edict_of_tolernace
+	}
+	
+	option = {
+		name = me_austria_missions.52.a
+		
+		add_country_modifier = {
+			name = me_hab_no_tolerance
+			duration = 7300
+		}
+	}
+	
+	option = {
+		name = me_austria_missions.52.b
+		
+		add_country_modifier = {
+			name = me_hab_edict_of_tolerance
+		}
```

Comment:
does this work or does it isappear after a month?

Reply:
Fuc